Source/WTF/wtf/text/cocoa/ContextualizedNSString.mm

```diff
@@ -67,9 +65,9 @@ - (void)getCharacters:(unichar *)buffer range:(NSRange)range
     auto contentsSubstring = contents.substring(contentsLow - context.length(), contentsHigh - contentsLow);
     static_assert(std::is_same_v<std::make_unsigned_t<unichar>, std::make_unsigned_t<UChar>>);
     contextSubstring.getCharacters(reinterpret_cast<UChar*>(buffer));
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
     contentsSubstring.getCharacters(reinterpret_cast<UChar*>(buffer) + contextSubstring.length());
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
```

Comment:
I presume we’ll fix this one when we fix `getCharacters` to take a `span` argument?

Reply:
Yes, this is a bigger refactoring so I kept it out of this PR.